functions/src/__tests__/authentication.test.ts

```diff
@@ -215,9 +216,9 @@ describe("useAuthentication", () => {
       uid: "abc123",
     } as admin.auth.DecodedIdToken;
 
-    fakeAuth.verifyIdToken.mockResolvedValue(decryptedToken);
+    fakeAuth.verifyIdToken.mockReturnValue(decryptedToken);
```

Comment:
@thomik-corp for some weird reason, after updating to use ESM, the `mockResolvedValue` insists on having a value type of `never`, so it does not let me pass in any return value. Switching to `mockReturnValue` fixes this for now, and doesn't affect the test (`await` simply has no effect). I'm not even sure how to track down this bug or how to report it.

If you are able to do some digging (or at least see if the behavior is replicated on your machine) that would be much appreciated. Otherwise I will leave as-is and just add a comm lent explaining what's going on.

<img width="659" alt="Screen Shot 2022-04-25 at 10 57 29 AM" src="https://user-images.githubusercontent.com/7362472/165116090-ff467a78-62d5-4de1-88c3-583db33e7ed6.png">

<img width="553" alt="Screen Shot 2022-04-25 at 10 58 28 AM" src="https://user-images.githubusercontent.com/7362472/165116099-19c26a2e-dd09-48a3-be72-635db1be4406.png">